src/documentation/SidebarContent/SidebarContent.stories.js

```diff
@@ -0,0 +1,86 @@
+import React, {useState} from 'react';
+
+import markdown from './README.mdx';
+import Sidebar from './Sidebar';
+import DATA from './PHONEBOOK.json'
+import User from '../../components/User';
+import {List, ListItem} from '../../components/List'
+import MainNavigation from '../../components/MainNavigation';
+
+export default {
+    title: 'Templates/SidebarContent',
+    parameters: {
+      status: 'released',
+      mdx: markdown,
+      previewWidth: 'full',
+    },
+  };
+
+export const Regular = (args) => (
+  <Sidebar />
+)
+
+
+export const Sidebarr = (args) => {
+  const [content, setContent] = useState(null);
+
+  const updateContent = (user) => {
+    const sidebarContent = (
+      <div style={{backgroundColor:'#fff', overflow:'scroll', padding:'1rem'}}>
```

Comment:
This part will be then replaced by the new `<Item />` component?

Reply:
Not sure of what you mean but the `sidebarContent` variable is what what will appear of the right side(content side). The item section is already in the sidebar logic. We can hope on a call so we go through it together

Reply:
The List content inside the sidebar should be a component which reusable for similar cases. I've made a commit here: https://github.com/wfp/ui/tree/feat/sidebar-content
For example in the Wiki https://dev.azure.com/worldfoodprogramme/ui/_wiki/wikis/ui.wiki/1827/Advanced-templates on the `School Connect Inventory List (Sidebar, Mobile)` screenshot there is the sidebar and a similar list in the content section. Both can be covered by this component.